cypress/e2e/shared/notificationEndpoints.test.ts

```diff
@@ -248,6 +272,94 @@ describe('Notification Endpoints', () => {
     cy.getByTestID(`endpoint-card--name ${name}`).should('not.exist')
   })
 
+  // to be used for status change values
+  const elementTextHasValueInRange = (
+    elem: JQuery,
+    min: number,
+    max: number
+  ) => {
+    const t = elem.text().match(/\d+/)
+    if (!t) {
+      throw `Element text: "${elem.text()}" contains no value`
+    }
+    expect(parseInt(t[0]))
+      .to.be.at.least(min)
+      .and.to.be.at.most(max)
+  }
+
+  it('can change endpoint status', () => {
+    cy.intercept('PATCH', '/api/v2/notificationEndpoints/*').as('patchEndp')
```

Comment:
I meant for all the aliases like this..

Reply:
Sorry, but these types of aliases are part of the code base going back two years and more.  It seems to me this is accepted practice in Influx.

Reply:
i understand that but I am not asking you to change it across the codebase. just here. practices evolve and so should we. rest is upto you, if you don't want to rename this.